src/Commands/TerminusCommand.php

```diff
@@ -41,4 +42,15 @@ protected function log()
     {
         return $this->logger;
     }
+
+    /**
+     * Override Robo's IO function with our custom style.
+     */
+    protected function io()
+    {
+        if (!$this->io) {
+            $this->io = new TerminusStyle($this->input(), $this->output());
+        }
+        return $this->io;
+    }
```

Comment:
@greg-1-anderson We could probably avoid even overriding this. Now that I am seeing it in a PR it occurs to me that we could just set `this->io and Robo will just use that one...perhaps I should refactor this to be less invasive...